composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
+        "php": "~5.3",
```

Comment:
for all coming to this page looking for a solution. a quick & dirty fix is using `--ignore-platform-reqs` on composer